db/migrate/20210515220031_backfill_selected_boolean_for_feedback_provided_references.rb

```diff
@@ -0,0 +1,5 @@
+class BackfillSelectedBooleanForFeedbackProvidedReferences < ActiveRecord::Migration[6.1]
+  def change
+    ApplicationReference.feedback_provided.update_all(selected: true)
```

Comment:
I figured that using update_all here to avoid touching the updated_at seems reasonable. Does that seem right?